app/subscriber/src/features/filter-media/FilterMedia.tsx

```diff
@@ -101,6 +101,7 @@ export const FilterMedia: React.FC<IFilterMediaProps> = ({ loaded }) => {
       <ContentListActionBar
```

Comment:
Maybe the ContentListActionBar should require the property to be provided?

Reply:
Ah never mind, we use that component when viewing singular pieces of content too, which won't have an `onClear`